lib/Froxlor/Validate/Validate.php

```diff
@@ -90,6 +110,9 @@ public static function validate_ip2($ip, $return_bool = false, $lng = 'invalidip
 			$ip_cidr = explode("/", $ip);
 			if (count($ip_cidr) == 2) {
 				$ip = $ip_cidr[0];
+				if(in_array((int)strlen((string)$ip_cidr[1]),array(1,2))) {
```

Comment:
same as in lib/Froxlor/Settings/Store.php:305 - better use `if (strlen($ip_cidr[1]) <= 2) {` and check for ipv6

Reply:
Changed